### PR TITLE
Replace target merchant box with banner

### PR DESCRIPTION
**Pull request by danthaman44**
Desktop:
Screen Shot 2020-09-05 at 10 15 53 PM

Mobile:
Screen Shot 2020-09-05 at 10 16 19 PM

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "footer": "100% of all proceeds go to the businesses, and we cover the credit card processing fees."
   },
   "descriptionBox": {
-    "header": "Meet our Target Merchant",
+    "header": "Our Target Merchant",
```
Comment (justintmckibben) [inline src/locales/en/translation.json:77]: Update the Chinese version too?

File: src/components/MerchantsPage/styles.module.scss [hunk 2]
```diff
@@ -140,6 +141,58 @@
...
+  grid-row: 1;
+  grid-column: 1;
+  background-color: #a8192e;
+  height: 240px;
+  width: 100vw;
+  position: relative;
+  left: 50%;
+  right: 50%;
+  margin-top: 100px;
+  margin-left: -50vw;
```
Comment (justintmckibben) [inline src/components/MerchantsPage/styles.module.scss:165]: This isn’t ideal

File: src/components/MerchantsPage/MerchantDescriptionBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+};
+
+export default MerchantDescriptionBanner;
+
+const List = styled.ul`
+  padding: 0 5px 0 20px;
+`;
+
+const Bullets = styled.li`
+  font-family: Open Sans;
```
Comment (justintmckibben) [inline src/components/MerchantsPage/MerchantDescriptionBanner.tsx:43]: We need backup fonts

**Review by justintmckibben — APPROVED** [review]

File: src/components/MerchantsPage/styles.module.scss (hunk 2 above)
Comment (danthaman44) [reply src/components/MerchantsPage/styles.module.scss:165]: ahh yeah agree. I copied this from the implementation of the NYC map. Do you know any other ways to achieve the full-screen width effect?

File: src/locales/en/translation.json (hunk 1 above)
Comment (danthaman44) [reply src/locales/en/translation.json:77]: Will do - I'll sync with Chel

File: src/components/MerchantsPage/MerchantDescriptionBanner.tsx (hunk 3 above)
Comment (danthaman44) [reply src/components/MerchantsPage/MerchantDescriptionBanner.tsx:43]: Good call, adding `'sans-serif' as a backup

**Pull request merged**